optuna/study.py

```diff
@@ -236,10 +236,9 @@ def optimize(
                 The number of parallel jobs. If this argument is set to :obj:`-1`, the number is
                 set to CPU counts.
             catch:
-                A study continues to run even when a trial raises one of exceptions specified in
-                this argument. Default is (`Exception <https://docs.python.org/3/library/
-                exceptions.html#Exception>`_,), where all non-exit exceptions are handled
-                by this logic.
+                A study continues to run even when a trial raises one of the exceptions specified
+                in this argument. Default is an empty tuple, i.e. the study will stop for any
```

Comment:
I think that "for any exception" is a bit misleading because Optuna catches `TrialPruned` exception even when `catch=()` is specified.

Reply:
You're absolute right. I made a slight change in https://github.com/pfnet/optuna/pull/638/commits/74267fffdbc4a2528cf632e32a078d999d04c22f.